### ISSUE TITLE
Touch target size is too small on mobile

### ISSUE THREAD

**Issue by superuser-does** [TiddlyWiki/TiddlyWiki5#9935]
## Problem

Touch target size is consistently too small on mobile viewports, making TiddlyWiki difficult to navigate on touch devices.

This is an accessibility and general UX issue. These are the best practice recommendations:

* [WCAG AA](https://www.w3.org/WAI/WCAG22/Understanding/target-size-minimum.html ): 24×24px – which **several parts of TiddlyWiki fail to meet**. See _Evidence_ below
* [WCAG AAA](https://www.w3.org/WAI/WCAG21/Understanding/target-size.html): 44×44px - which matches [Apple iOS guidelines](https://developer.apple.com/design/human-interface-guidelines/accessibility#Mobility)
*  [Chrome for Devs](https://web.dev/articles/accessible-tap-targets): 48×48px - which matches [Android guidelines](https://support.google.com/accessibility/android/answer/7101858?hl=en-GB)

There is a little more to these e.g. size can be smaller when a link is inline with text.
You can find a [good summary on Smashing Magazine](https://www.smashingmagazine.com/2023/04/accessible-tap-target-sizes-rage-taps-clicks/).

I want to especially highlight that *large commercial vendors are going for AAA and beyond*. I can say is extremely rare in all my time in the software industry, given that their hand is not forced by regulators.

That's why I encourage you to think of this as a broad and major UX issue, rather than solely a matter of accessibility.
They are going above the bare minimum of AA for a reason. The limit is probably too lax, based on Apple's and Google's (and other UX researchers') testing.

### Evidence

All screenshots from TiddlyWiki.com, which is reasonably close to vanilla.

The below are relatively common elements which a user can reasonably expect to come across either in their wiki, or a hosted wiki online - the most important of these being tiddlywiki-com.

| Element | Screenshot |
|--------|--------|
| PageControls<br>e.g. create tiddler button | <img height="700" alt="Image" src="https://github.com/user-attachments/assets/8732f2df-5be4-4df9-bdf5-337bc1ab07c1" /> |
| ViewToolbar button | <img height="700" alt="Image" src="https://github.com/user-attachments/assets/92b3f78a-e67a-4b95-931d-9b2d21726541" /> |
| Droppable list in Open tab | <img height="700" alt="Image" src="https://github.com/user-attachments/assets/ecf8063d-3150-4126-b5b5-95c17fdbbee0" /> | 
| Search button | <img height="700" alt="Image" src="https://github.com/user-attachments/assets/ae52572d-5d0f-4067-9191-105f2985801e" /> | 
| Search results | <img width="3234" height="1612" alt="Image" src="https://github.com/user-attachments/assets/a11591d4-c4ea-4bba-b13c-ba3b092becd1" /> |
| Alerts | <img height="500" alt="Image" src="https://github.com/user-attachments/assets/f8cb3fc0-c5cc-43ff-bbfd-465f1c0d3968" /> |
| Table of contents (shown here within the menubar) | <img height="700" alt="Image" src="https://github.com/user-attachments/assets/21ebb148-b45b-4008-8c27-89e311f1f7bc" /> |
| Menubar hamburger button | <img height="700" alt="Image" src="https://github.com/user-attachments/assets/0507703a-686a-4a26-aa74-c860b36f058b" /> |

## Solution

The default layout needs to have larger touch target sizes, at least for smaller viewports, and at least when using `coarse` pointers in CSS terms (which almost always means touch).

I believe TiddlyWiki should accept the risk that this would be a **breaking change** for the customisations many people have made.
In the long term, real mobile usability _needs_ to be in the default layouts.

## Alternatives considered

It could be introduced as a new layout, leaving the existing layout(s) as legacy options, if the breakage is too great.

## Additional context

Relates to:
* #9794
* #9802 - the new tab control of which ultimately inspired this, in the sense of showing openness to radically changing the default appearance of TW.

I believe my table of screenshots above covers most, if not all of the widgets/elements that should be addressed.

**Comment by Jermolene** [comment]
Hi @superuser-does I think these would be worthwhile improvements. As you note, the way to implement this is a new default theme.
